hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/NearCacheExecutor.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2008-2013, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.cache.impl.nearcache;
+
+/**
+ * Interface to support executor service for {@link com.hazelcast.cache.impl.nearcache.NearCache}
+ * to run their tasks.
+ */
+public interface NearCacheExecutor {
```

Comment:
Why not use `java.util.concurrent.Executor`? How is this interface different from `j.u.c.Executor`? 

```java
public interface Executor {
    void execute(Runnable command);
}
```